app/services/job_creator.rb

```diff
@@ -34,14 +34,14 @@ def self.process_payload(formatted_data)
 
   def self.create_job_from_found_company(formatted_data, company_id)
     company = create_company(formatted_data[:company][:name], company_id)
-    job = company.jobs.create!(title: formatted_data[:job][:title], description: formatted_data[:job][:description], url: formatted_data[:job][:url])
+    job = company.jobs.find_or_create_by!(title: formatted_data[:job][:title], description: formatted_data[:job][:description], url: formatted_data[:job][:url])
```

Comment:
@lsaville This is where it looked like the jobs were being created. I changed it to either return the existing job by title, description, and url or create it. Let me know what you think